fhirstore/search_engine.py

```diff
@@ -28,12 +65,12 @@ def current_url(self):
 
     def get_index_name(self):
         """ """
-        return "fhirstore"
+        return self.es_index
 
     def get_mapping(self, resource_type: str, index_config=None):
         """ """
         try:
-            return {"properties": self.mappings[resource_type]}
+            return self.mappings[resource_type]
```

Comment:
Did you also change this somewhere else?

Reply:
yes, in the generate_mappings method in fhirpath (see nazrulworld/fhirpath@cf0e518)